src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -31,19 +31,14 @@ On the opening page of **Cardinality Management**, you can view the **Usage over
     src="/images/apis-and-data_screenshot-crop_usage-over-time.webp"
 />
 
-This shows following:
-
-* The limit that is currently at 5 million account cardinality.
-* The current cardinality usage for the account, which is in the 2 to 4 million range. If there's no change in the value, then a new data point isn't reported. This can appear as an empty data point, but it's not data loss. This behavior may change as the feature is finalized.
-
-
 <Callout variant="tip">
-  You can modify this timeseries by changing the time-picker in the top right corner. Also, the time-picker only affects the timeseries and does not affect the **Metrics** table under the timeseries.
+  If the cardinality usage value remains unchanged, then a new data point isn't reported. So, you may see an empty data point. This isn't data loss.
 </Callout>
 
+To adjust the timeseries, use the time picker in the top right corner. This change only affects the timeseries in the chart and does't impact the Metrics table following it.
 ### Metrics table [#metrics-table]
+Check the top 20 metrics with the highest cardinality in the selected account for the day. View the breakdown of account cardinality in the <DNT>**Metrics**</DNT> table, organized by metric name.
 
-In the **Metrics** table, you can see the breakdown of account cardinality by metric name:
 
 <img
```

Comment:
Justify this image

Reply:
This image extends the previous one. On the Account Cardinality page, we have a chart followed by a table. I've separated them into two parts (first the chart, then the table) to explain each section clearly. The full image would be too large.